otter/cloud_client/__init__.py

```diff
@@ -680,7 +701,7 @@ def check_invalid_nodes(exc_info):
     ).on(
         error=_only_json_api_errors(
             lambda c, b: _process_clb_api_error(c, b, lb_id))
-    ).on(success=lambda _: None)
+    ).on(success=lambda _: raise_(partial) if partial is not None else None)
```

Comment:
I tend to prefer to put the positive case first, i.e.:

`None if partial is None else raise_(partial)`